examples/dcutr/src/main.rs

```diff
@@ -101,15 +102,11 @@ fn main() -> Result<(), Box<dyn Error>> {
         noise::NoiseAuthenticated::xx(&local_key)
             .expect("Signing libp2p-noise static DH keypair failed."),
     )
-    .multiplex(libp2p_yamux::YamuxConfig::default())
+    .multiplex(yamux::YamuxConfig::default())
     .boxed();
 
     #[derive(NetworkBehaviour)]
-    #[behaviour(
-        out_event = "Event",
-        event_process = false,
-        prelude = "libp2p_swarm::derive_prelude"
-    )]
+    #[behaviour(out_event = "Event", event_process = false, prelude = "derive_prelude")]
```

Comment:
```suggestion
    #[behaviour(out_event = "Event", event_process = false)]
```

When depending on the `libp2p` crate, this is not needed!